src/server/api.ts

```diff
@@ -27,6 +28,12 @@ type NdlaUser = {
   permissions?: string[];
 };
 
+const aiModelID = process.env.NDLA_AI_MODEL_ID;
+const aiRegion = process.env.NDLA_AI_MODEL_REGION;
+const aiSecretKey = process.env.NDLA_AI_SECRET_KEY;
+const aiSecretID = process.env.NDLA_AI_SECRET_ID;
+const transcriptionBucketName = process.env.S3_TRANSCRIPTION_BUCKET_NAME;
```

Comment:
Bruke samme navn overalt. Ref https://github.com/NDLANO/deploy/pull/1009
```suggestion
const transcriptionBucketName = process.env.TRANSCRIBE_FILE_S3_BUCKET;
```